sdk/appconfiguration/app-configuration/test/public/utils/testHelpers.ts

```diff
@@ -106,6 +97,17 @@ export async function deleteKeyCompletely(
   }
 }
 
+export async function deleteEverySetting(): Promise<void> {
```

Comment:
Doesn't look like this function is getting used.

Reply:
It seems like I didn't push he latest change I made because the goal is to use this function in the `after` section instead of having the duplicate code